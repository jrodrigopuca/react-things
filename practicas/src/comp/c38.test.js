import  React from 'react';
import C38 from './c38.jsx';
import renderer from 'react-test-renderer';

it('renderiza',()=>{
    const tree = renderer.create(<C38>Click me</C38>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});

