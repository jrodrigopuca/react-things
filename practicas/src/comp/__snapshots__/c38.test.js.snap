// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renderiza 1`] = `
<a
  href="#"
>
  Click me
</a>
`;
